shared/src/commonTest/kotlin/co/touchlab/kampkit/BaseTest.kt

```diff
@@ -1,7 +1,3 @@
 package co.touchlab.kampkit
 
-import kotlinx.coroutines.CoroutineScope
-
-expect abstract class BaseTest() {
-    fun <T> runTest(block: suspend CoroutineScope.() -> T)
-}
+expect abstract class BaseTest()
```

Comment:
As discussed in chat, we're only using this now to get robolectric into the Android tests. We should:

1. Create an expect/actual for `@RunWith`
2. Create an expect/actual for `AndroidJUnit4`
3. Only apply to needed classes (ex `SqlDelightTest`)